src/frontend/screens/ObservationEdit/index.tsx

```diff
@@ -57,25 +64,31 @@ export const ObservationEdit: NativeNavigationComponent<'ObservationEdit'> & {
     });
   }, [navigation, observationId]);
 
-  const handleCameraPress = React.useCallback(() => {
+  const handleCameraPress = useCallback(() => {
     navigation.navigate('AddPhoto');
   }, [navigation]);
 
-  const handleDetailsPress = React.useCallback(() => {
+  const handleDetailsPress = useCallback(() => {
     navigation.navigate('ObservationFields', {question: 1});
   }, [navigation]);
 
   const bottomSheetItems = [
     {
-      icon: <></>,
+      icon: <Audio width={30} height={30} />,
+      label: t(m.audioButton),
+      onPress: () => {},
+    },
```

Comment:
we should hide audio under a feature flag

Reply:
Okay 👍  I called that: `EXPO_PUBLIC_FEATURE_AUDIO`
@ErikSin maybe we have to create some `.env.example`?